src/main/java/org/killbill/billing/plugin/stripe/StripePaymentPluginApi.java

```diff
@@ -622,7 +622,10 @@ public HostedPaymentPageFormDescriptor buildFormDescriptor(final UUID kbAccountI
         String defaultCurrency = account.getCurrency() != null ? account.getCurrency().name() : "USD";
 
         Map<String, Object> params = new HashMap<String, Object>();
-
+        Map<String, Object> metadata = new HashMap<String, Object>();
+        customFields.forEach(p -> metadata.put(p.getKey(), p.getValue()));
+        params.put("metadata", metadata);
```

Comment:
You are sending a lot of information here (custom Kill Bill parameters but also everything that is part of the `line_items` for instance). Is that on purpose?

Reply:
I see I have duplicates in there. Added a filter to remove them.